featuretools/wrappers/sklearn.py

```diff
@@ -0,0 +1,261 @@
+                                   entityset=self.entityset,
+                                   target_entity=self.target_entity,
+                                   instance_ids=entity_ids,
+                                   agg_primitives=self.agg_primitives,
+                                   trans_primitives=self.trans_primitives,
+                                   allowed_paths=self.allowed_paths,
+                                   max_depth=self.max_depth,
+                                   ignore_entities=self.ignore_entities,
+                                   ignore_variables=self.ignore_variables,
+                                   seed_features=self.seed_features,
+                                   drop_contains=self.drop_contains,
+                                   drop_exact=self.drop_exact,
+                                   where_primitives=self.where_primitives,
+                                   max_features=self.max_features,
+                                   save_progress=self.save_progress,
+                                   features_only=True,
+                                   training_window=self.training_window,
+                                   approximate=self.approximate,
+                                   chunk_size=self.chunk_size,
+                                   n_jobs=self.n_jobs,
+                                   dask_kwargs=self.dask_kwargs,
+                                   verbose=self.verbose)
+        return self
+
+    def transform(self, entity_ids, y=None):
+        """Wrapper for calculate_feature_matix
+
+            Calculates a matrix for a given set of instance ids and calculation times.
+
+            Args:
+                entity_ids (list): List of instances to calculate features on.
+
+            See Also:
+                :func:`computational_backends.calculate_feature_matrix`
+        """
+        X_transformed =    calculate_feature_matrix(features=self.feature_defs,
+                                                    entityset=self.entityset,
+                                                    instance_ids=entity_ids,
+                                                    entities=self.entities,
+                                                    relationships=self.relationships,
+                                                    training_window=self.training_window,
+                                                    approximate=self.approximate,
+                                                    save_progress=self.save_progress,
+                                                    verbose=self.verbose,
+                                                    chunk_size=self.chunk_size,
+                                                    n_jobs=self.n_jobs,
+                                                    dask_kwargs=self.dask_kwargs,
+                                                    profile=self.profile)
+        return X_transformed
```

Comment:
you also need to pass through `y` in the case where it's not None.

also, make sure we have a test case for providing a `y` and not providing a `y`

Reply:
Hi @kmax12,

Finally, I have decided don't use 'y' argument in 'transform' method. It does not make sense. Let me know if you agree.